### DISCUSSION TITLE
Raspiblitz-img install on intel nuc 7i7 m2.sata 240GB

### DISCUSSION THREAD

**Discussion by DuckG** [raspiblitz/raspiblitz#4037] [Research]
I have a problem to use the image raspiblitz.

Etcher version: 1.18.11
Operating system and architecture: Intel NUC 7i7BNH   
Image flashed: raspiblitz-fat-v1.9.0-2023-05-22 also raspiblitz-v1.8.0c-2022-11-01.img
What do you think should have happened: Raspiblitz startup
What happened: non-blinking cursor on black screen.

Balena etcher(run as administrator) is completed image to m2.sata. no errors. 100% Successful. Flash completed!.
Hardware is good,(ssd is visable) i have tried other images (ie linux rockylinux or windows) 
SSD= 240Gb [kingston](m2.sata 22x80) = SM228053G2
![240GB-m2 sata](https://github.com/raspiblitz/raspiblitz/assets/37292164/9ae55b2b-d051-4429-a756-608c7d9ad283)

**Comment by adevoss** [comment]
Raspberry Pi has an arm cpu. Your NUC has a x86-64 cpu (amd64).

Two possibilities:
- use a raspiblitz amd64 image
- install Debian (11, bullseye!) and run the build_sdcard.sh script

https://github.com/raspiblitz/raspiblitz/blob/v1.9/alternative.platforms/README.md

**Comment by openoms** [comment]
As adevoss wrote above you need an amd64 image for PC, the RPi ones won't work.

The most up to date images and guide are in:
https://github.com/raspiblitz/raspiblitz/blob/dev/ci/README.md#images-generated-in-github-actions

**Comment by DuckG** [comment]
(thanks, and sorry for the late response)
hi, i am trying to download this image when i following this link. 
i can't find the newest raspiblitz-amd64-image.

I had previous older image months ago but i have only an shell. how can i start the setup raspiblitz?
